structure/0048-outreach-team.md

```diff
@@ -0,0 +1,71 @@
+# Outreach Team
+
+This document has been made according to the process outlined in [RFC
+0006][RFC6].
+
+## Summary
+
+The Outreach Team is a subteam of the Community Team in charge of outreach
+and communications within and outside the Quilt Project. This team is also
+responsible for working with the development teams to communicate the
+current status of development progress with the community and with any
+partners. In essence, it may be thought of as a "public relations
+department" of sorts.
+
+## Motivation
+
+This team is needed because developers would rather focus on development than
+on public relations, status updates, announcements, or team to-do lists.
+Historically, this has been managed by the Community Team as a whole.
+However, communications don't fall within the realm of the current Quilt
```

Comment:
This should say "Outreach" now because the name has changed.

Reply:
I'm fairly sure that's communications as in communications, not as in the Communications Team.

I mean, I guess you _could_ replace it with "outreach doesn't fall within[...]", but I feel like that's a bit nitpicky.

Reply:
Leaving this as "communications" would be better, since it links up with the next paragraph explaining what the Outreach Team is: "... will be responsible for all _communications_ outside of and within the Quilt project." _[emphasis mine]_

On another note regarding this paragraph, I think this last sentence needs a bit of a rewording; since the Outreach Team (which handles communications) is a subteam of the Community Team, then communications _do_ fall within the realm of the Community Team. In this case however, those responsibilities are delegated to a proper codified subteam with dedicated team members rather than managed by the Community Team's managers and moderator.